contracts/root/depositManager/DepositManager.sol

```diff
@@ -26,6 +26,14 @@ contract DepositManager is DepositManagerStorage, IDepositManager, IERC721Receiv
     _;
   }
 
+  modifier onlyDepositHelper() {
```

Comment:
How will it be a library?
We are calling `transferBatch` in the depositHelper. Bdw I am thinking of including these functions directly in the depositManager instead of having another contract